canonical-websites/www.ubuntu.com
# Remove grid tool

## Done

Remove grid tool

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- See that the grid toggle switch is no longer there


## Issue / Card
Fixes #3334
